brainscore/submission/configuration.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+import distutils
+
+from brainscore.submission.models import Model, Submission
+
+
+class BaseConfig:
+    def __init__(self, work_dir, jenkins_id, db_secret, config_path):
+        self.work_dir = work_dir
+        self.jenkins_id = jenkins_id
+        self.config_path = config_path
+        self.db_secret = db_secret
+        return
+
+
+class MultiConfig(BaseConfig):
+    def __init__(self, model_ids, **kwargs):
+        BaseConfig.__init__(self, **kwargs)
```

Comment:
is there a reason you're not using the `super` call? It doesn't matter in this case (yet), but if there is ever multiple inheritance, `super` guarantees calling every method exactly once (see e.g. https://stackoverflow.com/a/21639994/2225200)